### PR TITLE
Add monitoring with dask cluster docs

### LINKED ISSUE #3065: document prometheus support and available metrics
related to https://github.com/dask/distributed/pull/2499 and https://github.com/dask/distributed/pull/2833

I can not find prometheus metrics support and available metrics documented at [dask-distributed](https://distributed.dask.org/en/latest/search.html?q=prometheus&check_keywords=yes&area=default).

Happy to contribute a PR.

### PR DESCRIPTION
**Pull request by arpit1997**
fixes #3065 
cc: @jacobtomlinson

### CODE REVIEW

File: docs/source/monitoring-dask-cluster.rst [hunk 1]
```diff
@@ -0,0 +1,36 @@
+Monitoring dask cluster
```
Comment (mrocklin) [inline docs/source/monitoring-dask-cluster.rst]: For nowI recommend that we title this `Prometheus Monitoring` or something similarly specific.  There are many ways to monitor a Dask cluster, and if we use a more general title then we are obligated to cover all of them.

File: docs/source/index.rst [hunk 2]
```diff
@@ -112,6 +112,7 @@ Contents
    local-cluster
    ipython
    Joblib Integration <https://ml.dask.org/joblib.html>
+   monitoring-dask-cluster
```
Comment (mrocklin) [inline docs/source/index.rst]: Same here.  Given that this document currently only discusses Prometheus, I would prefer that it was named `prometheus`.

File: docs/source/monitoring-dask-cluster.rst (hunk 1 above)
Comment (arpit1997) [reply docs/source/monitoring-dask-cluster.rst]: resolved.

**Comment by mrocklin** [comment]
Thanks @arpit1997 !

Also, I notice that this is your first code contribution to this repository.  Welcome!

**Comment by arpit1997** [comment]
Thanks @mrocklin

**Pull request merged**